src/OWML.Logging/ConsoleUtils.cs

```diff
@@ -1,30 +1,27 @@
 ﻿using System;
-using System.Collections.Generic;
 using OWML.Common;
 
 namespace OWML.Logging
 {
 	public static class ConsoleUtils
 	{
-		private static readonly Dictionary<MessageType, ConsoleColor> MessageTypeColors = new Dictionary<MessageType, ConsoleColor> {
-			{ MessageType.Error, ConsoleColor.Red },
-			{ MessageType.Warning, ConsoleColor.Yellow },
-			{ MessageType.Success, ConsoleColor.Green },
-			{ MessageType.Message, ConsoleColor.Gray },
-			{ MessageType.Info, ConsoleColor.Cyan },
-			{ MessageType.Fatal, ConsoleColor.Magenta }
-		};
-
 		public static void WriteByType(MessageType type, string line)
 		{
 			if (string.IsNullOrEmpty(line))
 			{
 				return;
 			}
 
-			Console.ForegroundColor = MessageTypeColors.ContainsKey(type)
-				? MessageTypeColors[type]
-				: ConsoleColor.Gray;
+			Console.ForegroundColor = type switch
```

Comment:
Is it just me being used to older c# versions or is this less readable? :p

Reply:
especially with the discard _

Reply:
Maybe it's just you? :P I think it's great :D